FactoryWeather/App/VIP Home/HomeViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  HomeViewController.swift
+//  FactoryWeather
+//
+//  Created by Matej Korman on 08/01/2019.
+//  Copyright © 2019 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+protocol HomeDisplayLogic: class {
+    func displayWeather(_ weather: HomeDataSource, forLocation location: Location)
+    func searchTextFieldIsHidden(_ isHidden: Bool)
+}
+
+class HomeViewController: UIViewController {
+    var interactor: HomeBusinessLogic?
+    var router: HomeRoutingLogic?
+    
```

Comment:
I would remove this spacing as it's not really necessary. Nevertheless, we can leave it if you dig it.